lifecycle/lifecycle/deployer/redeploy.py

```diff
@@ -83,6 +83,51 @@ def reprovision_fatman(
         raise e
 
 
+def move_fatman(
+    fatman_name: str,
+    fatman_version: str,
+    new_infra_target: str,
+    config: Config,
+    plugin_engine: PluginEngine,
+    deployer_username: str,
+    auth_subject: Optional[models.AuthSubject],
+):
+    """Move fatman from one infrastructure target to another"""
+    fatman = read_fatman(fatman_name, fatman_version, config)
+    manifest = fatman.manifest
+    old_infra_target = fatman.infrastructure_target
+    assert fatman.manifest is not None, "fatman doesn't have Manifest data specified"
+    assert fatman.image_tag is not None, "fatman's image tag is unknown"
+    assert new_infra_target, "infrastructure target has to be specified"
+    assert new_infra_target != old_infra_target, "new infrastructure target has to be different from the current one"
+
+    deployment = create_deployment(manifest, deployer_username, new_infra_target)
+    try:
+        if manifest.secret_runtime_env_file:
+            fatman_secrets = _retrieve_fatman_secrets(config, manifest, plugin_engine, fatman)
+        else:
+            fatman_secrets = FatmanSecrets(git_credentials=None, secret_build_env={}, secret_runtime_env={})
+
+        with wrap_context('deploying fatman to a new infrastructure'):
+            provision_fatman(
+                config, manifest, fatman.image_tag, fatman_secrets.secret_build_env,
+                fatman_secrets.secret_runtime_env, deployment,
+                auth_subject, None, plugin_engine,
```

Comment:
```suggestion
                auth_subject, None, plugin_engine
```

Nitpick: Trailing comma

Reply:
Actually, I like trailing commas. This is my habit taken from Go, where it's obligatory. Sure, I can remove it in here.